src/openforms/forms/models/form_statistics.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from django.utils.formats import localize
+from django.utils.translation import gettext_lazy as _
+
+
+class FormStatistics(models.Model):
+    form = models.ForeignKey(
+        verbose_name=_("form"),
+        to="forms.Form",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+    )
+    form_name = models.CharField(
+        verbose_name=_("form name"),
+        max_length=150,
+        help_text=_(
+            "The name of the submitted form. This is saved separately in case of form deletion."
+        ),
+    )
```

Comment:
Question: does this get updated if the name of the form is changed?

Reply:
You are correct, this will be fixed.